testDictionaries/no-custom-fields.yml

```diff
@@ -429,6 +429,7 @@ fields:
 
   report:
     canUnpublishReports: true
+    titleMaxLength: 10
```

Comment:
Should be the same as in `anet-dictionary.yml` (other review suggestions sent separately).